spec/models/tag_spec.rb

```diff
@@ -103,5 +103,27 @@
         tag.destroy
       end
     end
+
+    it "syncs related elasticsearch documents" do
+      article = create(:article)
+      podcast_episode = create(:podcast_episode)
+      tag = described_class.find(article.tags.first.id)
```

Comment:
Here it is key that we are updating a `Tag` object and not a `ActsAsTaggableOn::Tag` object which is what `articles.tags.first` will return.